<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="PorteriaFutbol" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>620</width>
      <height>359</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/imgs/estadio-deportivo-ilustracion-porteria-futbol_53562-5609.jpg);</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>101</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/imgs/portero.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>290</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/imgs/balon.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="BotonInciar">
       <property name="text">
        <string>Iniciar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonDetener">
       <property name="text">
        <string>Detener</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonDetener_2">
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BotonDetener_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
